test/data/contracts/mock-atomicassets.ts

```diff
@@ -103,31 +103,33 @@ export namespace ActionParams {
         }
         export interface pair_string_ATOMIC_ATTRIBUTE {
             key: string
-            value: Types.pair_string_ATOMIC_ATTRIBUTE_value_variant
+            value: Types.variant_int8_int16_int32_int64_uint8_uint16_uint32_uint64_float32_float64_string_INT8_VEC_INT16_VEC_INT32_VEC_INT64_VEC_UINT8_VEC_UINT16_VEC_UINT32_VEC_UINT64_VEC_FLOAT_VEC_DOUBLE_VEC_STRING_VEC
+        }
+        export interface variant_int8_int16_int32_int64_uint8_uint16_uint32_uint64_float32_float64_string_INT8_VEC_INT16_VEC_INT32_VEC_INT64_VEC_UINT8_VEC_UINT16_VEC_UINT32_VEC_UINT64_VEC_FLOAT_VEC_DOUBLE_VEC_STRING_VEC {
+            value:
```

Comment:
This feels pretty unintuitive 😅 I'm guessing that the current code doesn't work with some variants on the atomicassets contract? I wonder if there is a way to have the contract automatically wrap the variant value in the correct format 🤔